packages/cli/src/bin/dev-portal/config.ts

```diff
@@ -38,6 +39,21 @@ export const configure = async (config: FrameworkConfigurator) => {
         });
     });
 
+    enableFeatureFlagging(config, (builder) => {
```

Comment:
add short comment that this is only for demo, needs to support by conf in future

Reply:
Done